docs/USAGE-GUIDE.md

```diff
@@ -386,39 +386,40 @@ s2n-tls uses pre-made security policies to help avoid common misconfiguration mi
-|          "20190122"                          |       |   X    |    X   |    X   |         |    X    |                   |   X   |    X    |  X   |     |  X  |   X   |                  |
-|        "default_tls13"                       |       |   X    |    X   |    X   |    X    |    X    |          X        |   X   |    X    |      |     |     |   X   |                  |
-|          "20190801"                          |       |   X    |    X   |    X   |    X    |    X    |          X        |       |    X    |      |     |     |   X   |                  |
-|          "20190802"                          |       |   X    |    X   |    X   |    X    |    X    |          X        |       |    X    |      |     |     |   X   |                  |
-|          "20200207"                          |       |   X    |    X   |    X   |    X    |    X    |          X        |       |    X    |      |     |     |       |                  |
-|          "20230317"                          |       |        |        |    X   |    X    |    X    |                   |   X   |    X    |      |     |     |   X   |                  |
-|          "rfc9151"                           |       |        |        |    X   |    X    |         |                   |   X   |    X    |      |     |  X  |   X   |                  |
-|  "CloudFront-TLS-1-2-2021"                   |       |        |        |    X   |    X    |         |          X        |   X   |    X    |      |     |     |   X   |                  |
-|  "CloudFront-TLS-1-2-2021-ChaCha20-Boosted"  |       |        |        |    X   |    X    |         |          X        |   X   |    X    |      |     |     |   X   |        X         |
-
-The "default" and "default_tls13" versions are special in that they will be updated with future s2n-tls changes and ciphersuites and protocol versions may be added and removed, or their internal order of preference might change. Numbered versions are fixed and will never change.
+|           version                            | SSLv3 | TLS1.0 | TLS1.1 | TLS1.2 | TLS1.3  | AES-CBC | AES-GCM | ChaCha20-Poly1305 | ECDSA | 3DES | RC4 | DHE | ECDHE | ChaCha20-Boosted |
+|----------------------------------------------|-------|--------|--------|--------|---------|---------|---------|-------------------|-------|------|-----|-----|-------|------------------|
+|          "default"                           |       |   X    |    X   |    X   |         |    X    |    X    |                   |       |      |     |     |   X   |                  |
+|        "default_tls13"                       |       |   X    |    X   |    X   |    X    |    X    |    X    |          X        |   X   |      |     |     |   X   |                  |
+|        "default_fips"                        |       |        |        |    X   |         |    X    |    X    |                   |   X   |      |     |  X  |   X   |                  |
```

Comment:
Since you undid all your deletions, do you want to create an issue to do that work? I think you were right and it's an important way to make this table more readable.